Sources/SPMBuildCore/Plugins/DefaultPluginScriptRunner.swift

```diff
@@ -466,12 +466,19 @@ public struct DefaultPluginScriptRunner: PluginScriptRunner, Cancellable {
 
         var env = Environment.current
 
+        // FIXME: This is largely a workaround for improper rpath setup on Linux. It should be
```

Comment:
issue (possibly-blocking) the path setup on linux is fixed in #9218.  This PR depend on it so it provides the proper fix/solution.

Reply:
The stdlib rpaths are a different set from those added in https://github.com/swiftlang/swift-package-manager/pull/9218. We can likely drop these runtime paths entirely soon but I need to test that out in another patch first